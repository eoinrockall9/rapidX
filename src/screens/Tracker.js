import React, { Component } from 'react';
import { StyleSheet, Dimensions, Button, View, Text } from 'react-native';


export default class Tracker extends Component {
  render() {
    return (
      <View style={styles.container}>
        {/* <MapView style={styles.mapStyle} /> */}
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
  mapStyle: {
    width: Dimensions.get('window').width,
    height: Dimensions.get('window').height,
  },
});
