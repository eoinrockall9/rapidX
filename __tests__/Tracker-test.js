import React from 'react';
import Tracker from '../src/screens/Tracker';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<Tracker />).toJSON();
    expect(tree).toMatchSnapshot();
})