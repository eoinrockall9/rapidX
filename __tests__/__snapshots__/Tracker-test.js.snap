// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <View
    style={
      Object {
        "backgroundColor": "white",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "center",
        "padding": 30,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 25,
          "marginBottom": 20,
          "textAlign": "center",
        }
      }
    >
      Add Distance in terms of KM
    </Text>
    <TextInput
      allowFontScaling={true}
      onChange={[Function]}
      rejectResponderTermination={true}
      style={
        Object {
          "borderColor": "black",
          "borderRadius": 8,
          "borderWidth": 1,
          "color": "black",
          "fontSize": 23,
          "height": 50,
          "marginRight": 5,
          "padding": 4,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>,
  <View
    style={
      Object {
        "backgroundColor": "white",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "center",
        "padding": 30,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 25,
          "marginBottom": 20,
          "textAlign": "center",
        }
      }
    >
      Add Time in minutes
    </Text>
    <TextInput
      allowFontScaling={true}
      onChange={[Function]}
      rejectResponderTermination={true}
      style={
        Object {
          "borderColor": "black",
          "borderRadius": 8,
          "borderWidth": 1,
          "color": "black",
          "fontSize": 23,
          "height": 50,
          "marginRight": 5,
          "padding": 4,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>,
  <View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "black",
          "borderColor": "white",
          "borderRadius": 8,
          "borderWidth": 1,
          "flexDirection": "row",
          "height": 45,
          "justifyContent": "center",
          "marginBottom": 10,
          "marginTop": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "alignSelf": "center",
            "color": "white",
            "fontSize": 18,
          }
        }
      >
        Add
      </Text>
    </View>
  </View>,
]
`;
